src/components/ConfirmationDialog.js

```diff
@@ -26,6 +27,7 @@ export default function ConfirmationDialog({
           variant="danger"
           ouiaId="confirm"
           onClick={() => onClose(true)}
+          isDisabled={selectedItems && selectedItems?.length === 0}
```

Comment:
This could be shortened to:

`isDisabled={selectedItems?.length === 0}`

Since the optional chaining takes cares of the case when selectedItems is 0.